packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -676,6 +686,24 @@ const Combobox = forwardRef<
             </span>
           )}
         </label>
+        <div>
+          {description && (
+            <span
+              className={classNames('Field__labelDescription', {
+                'Field__labelDescription--disabled': disabled
+              })}
+              id={descriptionId}
+            >
+              {description}
+            </span>
+          )}
+          {hasError && (
+            <div className="Error" id={errorId}>
```

Comment:
This needs to be updated based on: https://github.com/dequelabs/cauldron/blob/d985ed72b148d58a6031ddb52dd5aed5ecbbc426/packages/react/src/components/TextField/index.tsx#L120-L123

```suggestion
            <div className="Field__error" id={errorId}>
```